src/dune_engine/vcs.ml

```diff
@@ -62,8 +62,8 @@ let prog t =
 let run t args =
   let open Fiber.O in
   let+ s =
-    Process.run_capture ~display:!Clflags.display Strict (prog t) args
-      ~dir:t.root ~env:Env.initial
+    Process.run_capture ~display:Quiet Strict (prog t) args ~dir:t.root
+      ~env:Env.initial
```

Comment:
This is run by Dune for queries to the version control system. These are internal processes and do not need to be reported to the user.